README.md

```diff
@@ -41,10 +43,47 @@ cmake .. -DCMAKE_BUILD_TYPE=Release # use Debug for debug build
 make
 ```
 
-## Build knowledge base (from sc-machine/kb folder):
+## Build knowledge base
+
+This repo provides *build_kb.py* script to prepare and build knowledge base (KB). Note that you will need to run a server with a path to a compiled KB in the next step.
+
+### Usage example
 ```sh
-cd sc-machine/scripts
-./build_kb.sh
+cd sc-machine
+python3 scripts/build_kb.py ../ims.ostis.kb -c config/sc-machine.ini
```

Comment:
Can we implement add default config with listed arguments and run script without any arguments? I think that we wouldn't change this values to often

Reply:
Oh, I look at another PR to platform, as I understand bash scripts run python scripts like I suggest.